site/en/docs/android/custom-tabs/integration-guide/index.md

```diff
@@ -48,29 +49,84 @@ customTabsIntent.launchUrl(this, Uri.parse(url));
 
 By default, Custom Tabs launch as a full-window activity. Starting in Chrome 107, you can use partial Custom Tabs to specify a different launch height such that users can interact with your app while viewing web content. Users will be able to expand the Custom Tab to full-screen by dragging the toolbar handle up and restoring the initial launch height by dragging the handle down.
 
-{% Img src="image/6AZNJBRnkpQUWTKPzig99lQY8jT2/iMTvWNK7aEqqxHEHEXuv.png", alt="Example Partial Tab", width="584", height="1168" %}
+{% Img src="image/6AZNJBRnkpQUWTKPzig99lQY8jT2/iMTvWNK7aEqqxHEHEXuv.png", alt="Example Partial Tab", width="584", height="1168", class="screenshot" %}
+
+You can also choose to make the Custom Tab non-resizable, which prevents users from resizing it. If you set a value below 50% of the screen height in the intent, Chrome automatically adjusts the Custom Tab to 50% of the screen height. Custom Tabs are not supported when users are in multi-window or landscape mode.
+
+Partial Custom Tabs are supported by the AndroidX browser library from version 1.5 onwards:
 
-You can also choose to make the Custom Tab non-resizable. In this case, users will not have the ability to resize the Custom Tab. If a value below 50% of the screen height is provided in the intent, Chrome automatically adjusts the Custom Tab to 50% of the screen height. Custom Tabs are not supported when users are in multi-window or landscape mode.
+```groovy
+implementation 'androidx.browser:browser:1.5.0-alpha01'
+```
 
-The snippet below changes the launch height of the Custom Tab. `CustomTabsIntent.Builder` has a new method `setInitialActivityHeightPx()` that allows you to define the initial launch height in pixels. As a prerequisite to using Custom Tabs, you will need to [connect to the Custom Tabs Service](/docs/android/custom-tabs/integration-guide/#connect-to-the-custom-tabs-service) or launch the custom tabs intent using [Activity#startActivityForResult()](/reference/android/app/Activity#startActivityForResult(android.content.Intent,%20int)).
+To turn a Custom Tab into a partial Custom Tab, define the initial launch height in pixels by calling `CustomTabsIntent.Builder#setInitialActivityHeightPx()`. Furthermore, you need to either:
 
-There are properties that do not work when building an Intent for partial Custom Tabs:
```

Comment:
Delete this line too? (see comment farther down)

Reply:
this is removed